submissions/mxmgny/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Warcraft races</title>
+    <link rel="stylesheet" type="text/css" href="style\style.css">
+    <script src="script/script.js" async></script>
+</head>
+<body>
+
+</body>
+</html>
```

Comment:
Please add empty line at the end of every files

Reply:
Thank you for your attention! I resolve this problem